### PR TITLE
added documentations for getAmp, getType,getFreq,getPan

### PR DESCRIPTION
**Pull request by endurance21**
solves the issue 
#411
added  documentations for required METHODS
 
areas of impact 
src/oscillator.js

### CODE REVIEW

**Comment by endurance21** [comment]
@therewasaguy
first of all sorry, for the late response, as my sem exams were around so I couldn't make the pr soon.

i have added documentations, please see if something need to be added or so!
Thanks

File: src/oscillator.js [hunk 1]
```diff
@@ -309,6 +321,13 @@ define(function (require) {
   p5.Oscillator.prototype.setType = function(type) {
     this.oscillator.type = type;
   };
+  /**
+   *  Returns  current type of oscillator eg. 'sine', 'triangle', 'sawtooth' or 'square'.
+   *
+   *  @method  getType
+   *  @for p5.Oscillator
+   *  @param {String} type 'sine', 'triangle', 'sawtooth' or 'square'.
```
Comment (therewasaguy) [inline src/oscillator.js]: I think you meant `@returns`? This method doesn't accept a `@param` but those are the possible return values

File: src/oscillator.js [hunk 2]
```diff
@@ -213,6 +213,12 @@ define(function (require) {
   // these are now the same thing
   p5.Oscillator.prototype.fade =   p5.Oscillator.prototype.amp;
 
+  /**
+   * Returns the value of output gain 
+   *
+   *  @method  getAmp
+   *  @for p5.Oscillator
+   */
```
Comment (therewasaguy) [inline src/oscillator.js]: Should probably add a `@returns {number}` here, the description could be something like `Amplitude value between 0.0 and 1.0`

File: src/oscillator.js [hunk 3]
```diff
@@ -294,6 +300,12 @@ define(function (require) {
       return this.oscillator.frequency;
     }
   };
+   /**
+   * Returns the value of frequency of oscillator 
+   *
+   *  @method  getFreq
+   *  @for p5.Oscillator
+   */
```
Comment (therewasaguy) [inline src/oscillator.js]: a `@returns` would be helpful here

**Comment by endurance21** [comment]
@therewasaguy 
done and dusted !

File: src/oscillator.js [hunk 4]
```diff
@@ -213,6 +213,14 @@ define(function (require) {
   // these are now the same thing
   p5.Oscillator.prototype.fade =   p5.Oscillator.prototype.amp;
 
+  /**
+   * Returns the value of output gain 
+   *
+   *  @method  getAmp
+   *  @for p5.Oscillator
+   * 
+   * @return  {number} Amplitude value between 0.0 and 1.0
```
Comment (therewasaguy) [inline src/oscillator.js]: I think we need `@returns` with an "s" at the end :) https://jsdoc.app/tags-returns.html

File: lib/p5.sound.js [hunk 5]
```diff
@@ -1577,6 +1577,14 @@ var __WEBPACK_AMD_DEFINE_RESULT__;
 
 
   p5.Oscillator.prototype.fade = p5.Oscillator.prototype.amp;
+  /**
+   * Returns the value of output gain 
+   *
+   *  @method  getAmp
+   *  @for p5.Oscillator
+   * 
+   * @return  {number} Amplitude value between 0.0 and 1.0
```
Comment (therewasaguy) [inline lib/p5.sound.js]: the tag we need is `@returns` with an "s" at the end

File: src/oscillator.js (hunk 4 above)
Comment (endurance21) [reply src/oscillator.js]: okay ! i get , my bad

File: lib/p5.sound.js (hunk 5 above)
Comment (endurance21) [reply lib/p5.sound.js]: i think making change in src/oscillator.js is enough , after that this will be build by grunt ?

**Pull request merged**